src/content.js

```diff
@@ -63,17 +63,20 @@ function mapping(presented, callback) {
     }
 
     var doc = JSON.parse(body);
-    content_id = doc["content-id"];
-
-    logger.debug("Mapping service response: success => [" + content_id + "]");
-    callback(null, content_id);
+    logger.debug("Mapping service response: success => [" + doc + "]");
+    callback(null, doc);
   });
 }
 
 // Call the content service to acquire the document containing the metadata envelope and any
 // associated attributes at this content ID.
-function content(content_id, callback) {
-  var content_url = urljoin(config.content_service_url(), 'content', encodeURIComponent(content_id));
+function content(content_obj, callback) {
+  var content_url;
+  if (content_obj["proxy-to"]) {
+    content_url = content_obj["proxy-to"];
+  } else {
+    content_url = urljoin(config.content_service_url(), 'content', encodeURIComponent(content_obj["content-id"]));
+  }
```

Comment:
I'd personally put an `else if (content_obj["content-id"])` clause here, and add a final `else` that emits an Error to the callback, just to be safe.